renderer/index.js

```diff
@@ -795,6 +799,44 @@ function toggleSelectTorrent (infoHash) {
   update()
 }
 
+function openTorrentContextMenu (torrentSummary) {
+  var menu = new remote.Menu()
+  menu.append(new remote.MenuItem({
+    label: 'Save Torrent File As...',
+    click: () => saveTorrentFileAs(torrentSummary)
+  }))
+
+  menu.append(new remote.MenuItem({
+    label: 'Copy Instant.io Link to Clipboard',
+    click: () => clipboard.writeText(`https://instant.io/#${torrentSummary.infoHash}`)
+  }))
+
+  menu.append(new remote.MenuItem({
+    label: 'Copy Magnet Link to Clipboard',
+    click: () => clipboard.writeText(torrentSummary.magnetURI)
```

Comment:
Nevermind, you're right. It works, just not for the default torrents. Will fix that in a PR.